Lib/datetime.py

```diff
@@ -2539,7 +2508,7 @@ def _name_from_offset(delta):
     # Clean up unused names
     del (_DAYNAMES, _DAYS_BEFORE_MONTH, _DAYS_IN_MONTH, _DI100Y, _DI400Y,
          _DI4Y, _EPOCH, _MAXORDINAL, _MONTHNAMES, _build_struct_time,
-         _check_date_fields, _check_int_field, _check_time_fields,
+         _check_date_fields, _index, _check_time_fields,
```

Comment:
Looks like there's an effort to keep things in alphabetical order here; maybe move `_index` to the right place with respect to that order?